src/follow-me/style.scss

```diff
@@ -213,6 +213,81 @@ body.modal-open {
 	overflow: hidden;
 }
 
+/* Dark mode styles. */
+@media (prefers-color-scheme: dark) {
```

Comment:
I think the current code simply switches the global foreground color with the background and vice verse. This way we would not have to be so specific?